projects/sandbox/deployment/test/ingress.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
You can use the annotations and tls similarly to the ingress for metis ui.  
The metis-ui host/tls/secret `metis.test.eanadev.org` can in this case be perhaps `metis-sandbox.test.eanadev.org`.  
This applies to acceptance similarly as well.  
What do you think?